pnpm-lock.yaml

```diff
@@ -1,4 +1,8 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```

Comment:
Looks like your using pnpm v8.6.1, there was breaking change with this release which was fixed in v8.6.2. See https://github.com/pnpm/pnpm/issues/6648. If you upgrade pnpm and then upgrade ember-window-mock again then the lock file wont change it's lock file version. We want to remain on '6.0'. Everything else looks good

Reply:
Upgraded pnpm to v8.6.2 and installed the updated the package

Reply:
OK, I'd remove those settings. Also the build is failing for unrelated issues, it should pass if you rebase on main, I spent a bit of time yesterday merging outstanding PRs which fixed the failing build. Ideally this PR would be in two commits the first commit with the dependency update and the second with the feature update. Alternatively one commit would be fine, with all of them squashed. This means we'll have a clean history, and one which all commit should build and pass test suite, helpful if you ever need to git bisect the repo.

Reply:
Quick update, do want merge this PR, but ember-window-mock is a breaking change due to it dropping support for ember 4.4. So in another PR me or another maintainer will drop 4.4 and other unsupported ember versions. 4.4 last security updates are in a months time. The PR is also failing on other ember version but think its a CI issue, so I need to fix the build pipeline. Not sure of the timeline

Reply:
Thanks for the update @mrloop